### DISCUSSION TITLE
Is there a new version of JDK8?

### DISCUSSION THREAD

**Discussion by beibeiGeng** [ben-manes/caffeine#1540] [General]
The latest version is 2.9.3 of JDK8, It's been a few years since the last update,  Will it be updated of JDK8 in the future?

**Comment by ben-manes** [comment]
If there is a critical bug fix needed, but otherwise none is planned. There are performance, security, and maintainability reasons to upgrade, and it’s reaching end of life across the ecosystem. If you can make a strong argument then it will be considered.

**Reply by beibeiGeng** [reply]
Okay, thanks for letting me know.
